src/tests/getAttributes/getAttributes.test.ts

```diff
@@ -0,0 +1,216 @@
+import * as assert from 'assert';
```

Comment:
Last thing, can we please split this file into two or more so that we don't need to disable `max-file-line-count`